### ISSUE TITLE
Any ionic cli command gives "Bad project type: ionic-angular"

### ISSUE THREAD

**Issue by maxfloden** [ionic-team/ionic-cli#5002]
**Description:**

Type `ionic` (or any other ionic cli command) responds with `Bad project type: ionic-angular`
This for an Ionic3 / Angular6 project, works fine on other projects.

I have tried to downgrade to Ionic 6 cli but same error.
It seems to be related to Node version, I used to run project on Node v12 but then got an error that it reached end of life, switched to Node v14 but got same error:
`ERR: Your Node.js version is v14.21.3. Node.js 14 reached end-of-life on 2023-04-30 and is no longer supported. Please update to the latest Node LTS version.`
Then I switched to node v18 (have not tried node v16) and now instead get error `Bad project type: ionic-angular`

It worked fine when I worked on this project a few weeks ago and I do not believe that anything in environment has changed except that date got too close to Node v12/14 end of life.

**Steps to Reproduce:**

**Output:**

```
ionic
Bad project type: ionic-angular

```

**My `ionic info`:**

Cannot run ionic info. 
* Angular 6.1.10
* ionic-angular@3.9.10
* @ionic/cli@7.0.1

**Other Information:**

**Comment by maxfloden** [comment]
Possibly related to #5000

**Comment by maxfloden** [comment]
My bad. 
* I did in fact not try Ionic 6 as I stated. I thought I did: I installed it locally in project (npm i @ionic/cli@6) but global install (Ionic 7) was still being used for some reason (since no Ionic commands worked I could not check version).
* Ionic 1 and 3 support was removed in CLI v7, this what "Bad project type: ionic-angular" means (imho "Ionic v1 and v3 projects are no longer supported" would have been a clearer message) 
* After replacing global cli with v6 it works
